leaderboard/service/total_pages_test.go

```diff
@@ -0,0 +1,61 @@
+package service_test
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/topfreegames/podium/leaderboard/database"
+	"github.com/topfreegames/podium/leaderboard/service"
+)
+
+var _ = Describe("Service TotalPages", func() {
+	var ctrl *gomock.Controller
+	var mock *database.MockDatabase
+	var svc *service.Service
+
+	var leaderboard string = "leaderboardTest"
+	var totalMembers int = 11
+	var pageSize int = 10
+	var pageSize2 int = 3
+
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
+		mock = database.NewMockDatabase(ctrl)
+
+		svc = &service.Service{mock}
+	})
+
+	AfterEach(func() {
+		ctrl.Finish()
+	})
+
+	It("Should return number of pages if all is OK", func() {
+		mock.EXPECT().GetTotalMembers(gomock.Any(), gomock.Eq(leaderboard)).Return(totalMembers, nil)
+		mock.EXPECT().GetTotalMembers(gomock.Any(), gomock.Eq(leaderboard)).Return(totalMembers, nil)
+
+		pages, err := svc.TotalPages(context.Background(), leaderboard, pageSize)
+		Expect(err).NotTo(HaveOccurred())
+
+		calculatedPages := int(math.Ceil(float64(totalMembers) / float64(pageSize)))
+
+		Expect(pages).To(Equal(calculatedPages))
```

Comment:
As this is the same formula you used inside your implementation, it won't fail if the formula is wrong. I rather calculate by hand and then use the result I got. 
```suggestion
		Expect(pages).To(Equal(2))
```

Reply:
I always avoid using absolutely values, but in this case, make sense